openedx_proversity_reports/settings/production.py

```diff
@@ -87,3 +87,12 @@ def plugin_settings(settings):
         'OPR_GOOGLE_ANALYTICS_CREDENTIALS',
         settings.OPR_GOOGLE_ANALYTICS_CREDENTIALS
     )
+
+    settings.OPR_TIME_BETWEEN_SESSIONS = getattr(settings, 'ENV_TOKENS', {}).get(
```

Comment:
If this settings are for the Ginkgo-Hawthorn instance, the production file does not exist, so please apply the settings in the aws file. @andrey-canon

Reply:
check the aws file

Reply:
You right, I didn't remember.